metrics.go

```diff
@@ -165,3 +190,11 @@ func parseArchiveMetrics(data map[string]fronius.InverterArchive) {
 		siteMPPTVoltageGaugeVec.WithLabelValues(key, "2").Set(inverter.Data.VoltageDCString2.Values["0"])
 	}
 }
+
+func parseSmartMeterMetrics(data map[string]fronius.MeterData) {
+	log.WithField("meterData", data).Debug("Parsing data.")
+	for key, meter := range data {
+		meterEnergyRealSumConsumedVec.WithLabelValues(key).Set(meter.EnergyRealSumConsumed + config.Symo.OffsetConsumed)
+		meterEnergyRealSumProducedVec.WithLabelValues(key).Set(meter.EnergyRealSumProduced + config.Symo.OffsetProduced)
```

Comment:
I don't know the smart meter data...
Is this offset actually required or important here? What is the semantic of the offset? Or is this more relevant to your personal use case?

Reply:
see above